*** myspell_checker.cpp.orig	Thu Aug 20 20:58:44 2009
--- myspell_checker.cpp	Thu Aug 20 21:30:42 2009
***************
*** 373,385 ****
  
  	s_buildHashNames (names, broker, tag);
  
! 	for (size_t i = 0; i < names.size (); i++) {
  		if (g_file_test(names[i].c_str(), G_FILE_TEST_EXISTS)) {
  			if(s_hasCorrespondingAffFile(names[i])){
  				return g_strdup (names[i].c_str());
  			}
  		}
! 	}
  	
  	std::vector<std::string> dirs;
  	s_buildDictionaryDirs (dirs, broker);
--- 373,385 ----
  
  	s_buildHashNames (names, broker, tag);
  
! 	{for (size_t i = 0; i < names.size (); i++) {
  		if (g_file_test(names[i].c_str(), G_FILE_TEST_EXISTS)) {
  			if(s_hasCorrespondingAffFile(names[i])){
  				return g_strdup (names[i].c_str());
  			}
  		}
! 	}}
  	
  	std::vector<std::string> dirs;
  	s_buildDictionaryDirs (dirs, broker);
